applications/launchpad/gui-react/src/containers/MiningContainer/MiningBox/index.tsx

```diff
@@ -36,7 +38,7 @@ const parseLastSessionToCoins = (
         lastSession.total && lastSession.total[coin]
           ? lastSession.total[coin]
           : '0',
-      loading: !anyNonZeroCoin,
+      loading: !anyNonZeroCoin && theCurrentStatus === MiningBoxStatus.Running,
```

Comment:
Fix: show loader only if amount is not Zero and only in `Running` state.
It checks if any coin is not zero, ie. in merged mining we have xtr and xmr.